src/parser/base.rs

```diff
@@ -163,7 +162,7 @@ impl<'a> Parser<'a> {
         let value = if let Some(quote) = self.stream.expect_oneof_and_skip(&[b'"', b'\'']) {
             self.read_to(quote)
         } else {
-            self.read_to4([b' ', b'\n', b'/', b'>'])
+            self.read_to3([b' ', b'\n', b'>'])
```

Comment:
Per MDN, if the `/` isn't separated from the unquoted value by whitespace, it should be treated as part of the value. So I think it's correct to just omit it entirely here.

Reply:
See: https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags